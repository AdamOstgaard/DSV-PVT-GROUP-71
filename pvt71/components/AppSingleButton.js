import React from "react";
import { StyleSheet, TouchableOpacity, Text } from "react-native";

export default class AppSingleButton extends React.Component {
  render() {
    return (
      <TouchableOpacity style={{...styles.button, ...this.props.style}} onPress={this.props.onPress}>
      <Text style={{...styles.textStyle,...this.props.textStyle}}>{this.props.title}</Text>
    </TouchableOpacity>
    );
  }
}

const styles = StyleSheet.create({
  button: {
    backgroundColor: "#0d6f73",
    width: "100%",
    height: 60,
  },
  textStyle: {
    textAlign: "right",
    fontSize: 30,
    color: "#fff",
    paddingRight: 10,
    paddingTop: 8
  }
});