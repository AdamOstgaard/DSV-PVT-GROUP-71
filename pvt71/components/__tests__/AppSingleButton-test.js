import "react-native";
import React from "react";
import  AppSingleButton  from "../AppSingleButton";

import renderer from "react-test-renderer";

describe("AppSingleButton Test", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<AppSingleButton title='Next'/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});