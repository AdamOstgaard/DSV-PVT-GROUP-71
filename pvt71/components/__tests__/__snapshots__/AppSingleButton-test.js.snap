// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppSingleButton Test AppSingleButton Snapshot renders correctly 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "#0d6f73",
      "height": 60,
      "opacity": 1,
      "width": "100%",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#fff",
        "fontSize": 30,
        "paddingRight": 10,
        "paddingTop": 8,
        "textAlign": "right",
      }
    }
  >
    Next
  </Text>
</View>
`;
